Cakefile

```diff
@@ -166,6 +169,30 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
   console.log "built ... running browser tests:"
   invoke 'test:browser'
 
+task 'build:inline_runtime', 'build the inline iced3 runtime', ->
+  runtime_dir = path.dirname require.resolve 'iced-runtime-3'
+  code = ''
+  for name in ['const', 'runtime', 'library', 'main']
+    code += """
+      require['./#{name}'] = (function() {
+        var exports = {}, module = {exports: exports};
+        #{fs.readFileSync "#{runtime_dir}/#{name}.js"}
+        return module.exports;
+      })();
+    """
+  code = """
+    (function() {
+      function require(path){ return require[path]; }
+      #{code}
+      return require['./main'];
+    }());
+  """
+
+  unless process.env.MINIFY is 'false'
+    {code} = require('uglify-js').minify code, fromString: true
+
```

Comment:
Agreed all around.  Probably the most useful workflow is for us to output ES6, and then for consumer of the output code to minify and transpile as they see fit. Meaning, the whole browser operation of CoffeeScript might be moot for us, or more charitably, best-effort.